src/lib/string_utils.adb

```diff
@@ -0,0 +1,119 @@
+pragma Ada_2012;
+
+package body String_Utils
+with SPARK_Mode
+is
+
+   ------------
+   -- Length --
+   ------------
+
+   function Length (
+                    C_Str : System.Address;
+                    Max_Length : Natural := Natural'Last
+                   ) return Integer
```

Comment:
Could please adapt the style, so that it matches the first or the second example in the [Ada Style Guide](https://en.wikibooks.org/wiki/Ada_Style_Guide/Source_Code_Presentation#More_on_Alignment)? That are the styles which are usually used in the runtime.